test/asset-swapper/quote_consumer/feature_rules/transform_erc20_rule_test.ts

```diff
@@ -43,7 +43,7 @@ const expect = chai.expect;
 const { NULL_ADDRESS } = constants;
 const { MAX_UINT256, ZERO_AMOUNT } = contractConstants;
 
-describe('TransformERC20Rule', () => {
+describe.only('TransformERC20Rule', () => {
```

Comment:
did you mean to leave this?

Reply:
no 🤣  (good that eslint now catches this)